src/scores/utils.py

```diff
@@ -59,102 +50,14 @@ def gather_dimensions(  # pylint: disable=too-many-branches
     fcst_dims: Iterable[Hashable],
     obs_dims: Iterable[Hashable],
     *,  # Force keywords arguments to be keyword-only
+    weights_dims: Optional[Iterable[Hashable]] = None,
     reduce_dims: Optional[FlexibleDimensionTypes] = None,
     preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    score_specific_fcst_dims: Optional[FlexibleDimensionTypes] = None,
```

Comment:
The docstring needs updating to match the arg name.

Reply:
Also need to update the arg name twice further down in the docstring under the "Raises section"

Reply:
thanks, done